go.mod

```diff
@@ -65,6 +65,7 @@ require (
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 )
 
+// When updating replace rules, make sure to also update the rules in integration/client/go.mod
```

Comment:
If we need to do this for `integration/client`, does that mean any other random client importing containerd/containerd needs to copy over these `replace` rules too?

Reply:
Yes; replace rules are only respected locally, and ignored everywhere else (they were designed for local debugging only), so every consumer must copy those rules; which is why we should try to get rid of them

Reply:
Buildkit at some point had (I think deliberately) versions that didn't compile unless overridden for that reason (which was needed because go mod wasn't able to resolve the correct version and it broke several times) https://github.com/moby/buildkit/pull/1425 and issues linked from/to that